src/components/dataset/details/DatasetDetailView.jsx

```diff
@@ -32,20 +39,58 @@ const styles = (theme) => ({
   mainColumn: {
     display: 'flex',
     flexDirection: 'column',
-    paddingLeft: '0.5rem',
+    marginLeft: 40,
   },
-  relationshipsArea: {
+  snapshotsArea: {
     flexGrow: 1,
+    marginTop: '1.5rem',
   },
   spacer: {
     height: '4rem',
   },
   pageTitle: {
-    margin: '0.5rem',
-    fontWeight: theme.typography.bold,
+    marginBottom: '1rem',
+  },
+  snapshotCardsContainer: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(auto-fill, minmax(360px, 32%))',
+    gridGap: '1rem',
   },
 });
 
+const fakeSnapshots = [
```

Comment:
my gut reaction is that I want this in a file exclusively for testing and not here---although it looks like maybe you are planning to remove this entirely

Reply:
yep, going to be removed once we get an endpoint for this data